webapp/views/wpt-app.js

```diff
@@ -112,13 +112,16 @@ class WPTApp extends PathInfo(WPTFlags(TestRunsUIBase)) {
       <template is="dom-if" if="[[resultsTotalsRangeMessage]]">
         <info-banner>
           [[resultsTotalsRangeMessage]]
+          <template is="dom-if" if="[[!editable]]">
+            <a href="" onclick="[[loadDefaultProductURL]]"> (switch to the default product set instead)</a>
```

Comment:
```suggestion
            <a href="javascript:window.location.search='';"> (switch to the default product set instead)</a>
```

You're fighting against `app-location` here (see the [docs](https://www.webcomponents.org/element/@polymer/app-route/elements/app-location#integrating-with-other-routing-code:~:text=Integrating%20with%20other%20routing%20code)). You don't need to have an onclick listener for this though.

Reply:
And if you take this suggestion, remember to delete `loadDefaultProductURL` as well.